NEWS.md

```diff
@@ -1,12 +1,13 @@
-# ggExtra 0.7.1
+# ggExtra 0.8
 
 NEW FEATURES
 
-- Added support for violin plots
+- Added support for violin plots (#62)
+- Added support for mapping colour from the scatter plot to colour/fill in the marginal plots (#61)
 
 BUX FIXES
 
-- Make sure marginal data comes from correct data frame (#68) 
+- Make sure marginal data comes from correct data frame (#67) 
```

Comment:
I changed this to note the issue number, not the PR number. Is that the correct way to do it?

Reply:
Sure, either works. I think moving forward I'll try to be more consistent and robust by doing "Fixed bug #ISUSUENUMBER: make sure marginal data comes from correct data frame (#PRNUMBER)"